.github/workflows/test.yml

```diff
@@ -177,19 +175,16 @@ jobs:
           df -h
 
       - name: Install stable toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: stable
-          target: wasm32-unknown-unknown
-          components: rust-src
+        run: |
+          rustup install ${{ env.RUST_STABLE_VERSION }}
+          rustup default ${{ env.RUST_STABLE_VERSION }}
+          rustup target add wasm32-unknown-unknown
+          rustup component add rust-src
 
       - name: Install nightly toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: nightly
-          target: wasm32-unknown-unknown
+        run: |
+          rustup toolchain install "nightly-${{ env.RUST_NIGHTLY_VERSION }}" --profile minimal --component rust-src
+          rustup target add wasm32-unknown-unknown --toolchain "nightly-${{ env.RUST_NIGHTLY_VERSION }}"
```

Comment:
Not sure if we even need nightly for this job as it only uses `cargo test`?

Reply:
It seems we don't, removed in https://github.com/polkadot-fellows/runtimes/pull/247/commits/9373fde97864b53aff55bee83d01d7e47e7160c3